src/screens/Messages/Conversation/MessagesList.tsx

```diff
@@ -175,3 +177,26 @@ export function MessagesList() {
     </KeyboardAvoidingView>
   )
 }
+
+function useKeyboardVerticalOffset() {
+  const {top: topInset} = useSafeAreaInsets()
+  const [screenWindowDifference, setScreenWindowDifference] = React.useState(
+    () => Dimensions.get('screen').height - Dimensions.get('window').height,
+  )
+
+  useEffect(() => {
+    const subscription = Dimensions.addEventListener(
+      'change',
+      ({screen, window}) => {
+        setScreenWindowDifference(screen.height - window.height)
+      },
+    )
+    return () => subscription.remove()
+  }, [])
+
+  return Platform.select({
+    ios: topInset,
+    android: screenWindowDifference,
+    default: 0,
+  })
```

Comment:
i remember us having a problem with a `Platform.select` before in prod but can't remember what exactly it ended up being linked to. Do you remember?